go/filemanager/pfsmodules/command.go

```diff
@@ -49,8 +49,28 @@ func checkUser(pathStr string, user string) error {
 	return nil
 }
 
-// ValidatePfsPath returns whether a path is a pfspath.
-func ValidatePfsPath(paths []string, userName string) error {
+func isPublic(pathStr string) bool {
+	pathStr = path.Clean(strings.TrimSpace(pathStr))
+	a := strings.Split(pathStr, "/")
+
+	if len(a) >= 4 && a[3] == "public" {
+		return true
+	}
+
+	return false
+}
+
+func checkPublic(cmdName string) error {
+	switch cmdName {
+	case "ls", "stat":
```

Comment:
This limits the CLI to do only `ls` and `stat`, but does the backend still allow cmd like `rm` and `cp` to function? If so, user could easily patch here and run commands like `rm` to the public dataset.